.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       uses: actions/checkout@v2
     - name: minitest
       run: |
+        bundle install
+        rubocop
```

Comment:
This should probably be prefixed with `bundle exec`, no? Same actually goes for the command below.

Reply:
Thanks for pointing that out! I'm a total Ruby noob, so this is really helpful.

Reply:
🤗